test/classes/user.py

```diff
@@ -0,0 +1,170 @@
+'''
+Copyright 2017, DellEMC LLC
+
+     Author: Torrey Cuthbert
+   Filename: user.py
+Create Date: 04/09/2017
+Description: Defines a RackHD User account base class and methods. All account role types (Administrator, Operator, and
+             ReadOnly) inherit from this class. An instance of this class allows for user account creation and maintenance e.g.,
+             account password change.
+'''
+
+import fit_common
+import json
+import requests
+from requests.packages.urllib3.exceptions import InsecureRequestWarning
+requests.packages.urllib3.disable_warnings(InsecureRequestWarning)
+
+class User:
+    num_users = 0
+    timeout = 5
+
+    def __init__(self, username, passwd, role, redfish=False):
+        self.username = username
+        self.passwd = passwd
+        self.role = role
+        if fit_common.AUTH_TOKEN == 'Unavailable' or fit_common.REDFISH_TOKEN == 'None':
+            if not fit_common.get_auth_token():
+                if fit_common.VERBOSITY >= 4:
+                    print 'error, unable to obtain fit_common RackHD and Redfish API administrative tokens'
+                return None
+        # create a Redfish or RackHD account first (redfish=False) depending on the truth of redfish
+        # this logic allows the ability to test that an account for each endpoint gets created no matter which
+        # one is created first. the constructor does this basic check.
+        if not redfish:
+            if not self.createRackHDUser():
+                return None
+        else:
+            if not self.createRedfishUser():
+                return None
+        User.num_users += 1
+
+    def createRackHDUser(self):
+        headers = {
+            'content-type': 'application/json',
+            'accept': 'application/json'
+        }
+        payload = {
+            'username': self.username,
+            'password': self.passwd,
+            'role': self.role
+        }
+        url = "https://{0}:{1}/api/2.0/users?auth_token={2}".format(fit_common.fitargs()['rackhd_host'],
```

Comment:
think you can use fitcfg() instead of fitargs() for rackhd_host.  (args would be specific to command line opts)